src/cljs/zetawar/views.cljs

```diff
@@ -384,7 +384,12 @@
                                  :on-hide hide-settings}
      [:> js/ReactBootstrap.Modal.Header {:close-button true}
       [:> js/ReactBootstrap.Modal.Title
-       (translate :configure-faction-title)]]
+       (str "Configure Faction: "
```

Comment:
I'm not totally consistent yet, but I'd like to use it for all user visible strings. The reason is two fold. First, it gets the strings out of the UI. I think that's generally cleaner (text changes are separated from UI structure changes). Second, it lets us support multiple languages is the future.